Day_20/scoreboard.py

```diff
@@ -14,18 +14,28 @@ def __init__(self):
         self.penup()
         self.speed('fastest')
         self.hideturtle()
-        self.counter = 0
+        self.current_score = 0
+        self.high_score = 0
         self.goto(x=SCOREBOARD_XCOR, y=SCOREBOARD_YCOR)
+
+        self.get_highest_score()
         self.scoreboard_text()
 
+    def get_highest_score(self):
+        with open(file='./highest_score.txt', mode='r', encoding='utf-8') as f:
+            self.high_score = f.read()
```

Comment:
high score should be int when it is loaded, as the value can be used at many places that will lead to multiple type casting. like below where you are comparing it.

Reply:
Agreed. Raising a new pull request with correction